### DISCUSSION TITLE
So many changes...

### DISCUSSION THREAD

**Discussion by Demolutionnn** [community/community#22636] [New to GitHub]
<p>Hello. I have a strange problem.</p>
<p>I am using GitHub Extension for Visual Studio. After i clone master branch at the “Changes”  option in VS i see over 25 thousands changes. I guess those are the changes since the beginning of the project?</p>
<p>My version is identical as the repo. Am i supposed to see 0 changes? My question is: How i can clear those changes to show 0 without affecting the history in GitHub ( to clear just localy on my computer)</p>
<p>Thanks in advance!</p>

**Comment by jp2masa** [comment, marked as answer]
<p>That’s really weird…</p>
<p>Try checking the current status of the repo:</p>
<pre><code>git status
</code></pre>
<p>If it shows the changes, then you may want to remove them.</p>
<p>To remove all local changes, you can use:</p>
<pre><code>git clean -xdf
</code></pre>

**Comment by Demolutionnn** [comment]
<p>After i do the command :</p>
<pre><code>git clean -xdf
</code></pre>
<p> My changes are cleared it shows 0, but its just untill i restart Visual Studio and its showing 23 000 changes again … :frowning:</p>
<p>Maybe i am missing some finishing command? </p>

**Comment by mpboom** [comment]
<p>It might have something to do with permissions. For example, Atom (the text editor) also displays file permission changes as changes. Try to check if there is somewhere where the permissions of files may change.</p>

**Comment by tracinaho** [comment]
<p>This is true…is safe zoo bot a part of this?</p>
